src/database/migrations/20231219073900-role_permission_mapping.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('role_permission_mapping', {
+			role_id: {
+				allowNull: false,
+				type: Sequelize.INTEGER,
+				primaryKey: true,
+			},
+			permission_id: {
+				allowNull: false,
+				type: Sequelize.INTEGER,
+				primaryKey: true,
+			},
+			module: {
+				allowNull: false,
+				type: Sequelize.STRING,
+			},
+			actions: {
+				allowNull: false,
+				type: Sequelize.ARRAY(Sequelize.STRING),
+			},
+			created_at: {
+				allowNull: false,
+				type: Sequelize.DATE,
+			},
+			updated_at: {
```

Comment:
@sumanvpacewisdom - we should add created_by and updated_by as well which should be set to who is creating and updating these data.